HHS/Head-Start-TTADP
# [TTAHUB-3513] Al ttahub 3513 monitoring cron job

## Description of change

This runs a CRON job after the monitoring data is brought in via a post processing step. 

**NOTE: Before merging  we MUST update the start cut off date.**

## How to test

- Review the code.
- Manually run the job via the CLI and ensure we are creating the correct goals.

Criteria to keep in mind for when testing if a Monitoring Goal should be created:
1. **MonitoringReviews.reportDeliveryDate** is between the CUTOFF DATE (TBD) and TODAY
1.  **MonitoringReviewStatuses.name** = 'Complete'
2.  **MonitoringFindingStatuses."name"** = 'Active'
3. The **MonitoringReviews.reviewType** is one of the following:  'AIAN-DEF', 'RAN', 'Follow-up', 'FA-1', 'FA1-FR', 'FA-2', 'FA2-CR', 'Special'
4. There is a case were if the above criteria is met and there is currently a CLOSED monitoring goal for the grant we should set its status to 'Not started'.
5. Verify all fields on the newly created goals are correct

By running the first part of the query via a query analyzer you can get a count of how man goals you expect to have created.

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3513


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
